frontend/src/app/[locale]/search/error.tsx

```diff
@@ -51,8 +53,25 @@ function createBlankParsedError(): ParsedError {
   };
 }
 
-export default function SearchError({ error }: ErrorProps) {
+export default function SearchError({ error, reset }: ErrorProps) {
   const t = useTranslations("Search");
+  const searchParams = useSearchParams();
+  const previousSearchParams =
+    usePrevious<ReadonlyURLSearchParams>(searchParams);
+
+  useEffect(() => {
+    if (
+      reset &&
+      previousSearchParams &&
+      searchParams.toString() !== previousSearchParams?.toString()
+    ) {
+      reset();
+    }
+  }, [searchParams, reset]);
+
+  useEffect(() => {
+    console.error(error);
+  }, [error]);
```

Comment:
Could we use or extend one of the errors from https://github.com/HHS/simpler-grants-gov/blob/main/frontend/src/errors.ts so we have more consistent error formatting and handling?

Reply:
We could punt to this https://github.com/HHS/simpler-grants-gov/issues/3651 if that is what yr thinking

Reply:
the errors thrown here largely will be based on the definitions in our errors file, so not sure what to do here

Reply:
I would think we would wrap the output. I guess I don't see how this error is formed or called.